src/godot_fmod.h

```diff
@@ -161,7 +163,9 @@ namespace godot {
 
         ~Fmod();
 
-        static void _register_methods();
+        enum {
+            DECLARE_ALL_CONSTANTS
```

Comment:
Since we can declare constants properly now, I am doing something similar to the GDExtension demo project here.

Constants will effectively resolve here like so:
```cpp
enum {
GD_FMOD_INIT_3D_RIGHTHANDED = 0x00000004,
GD_FMOD_INIT_CHANNEL_DISTANCEFILTER = 0x00000200,
GD_FMOD_INIT_CHANNEL_LOWPASS = 0x00000100,
GD_FMOD_INIT_GEOMETRY_USECLOSEST = 0x00040000,
GD_FMOD_INIT_MIX_FROM_UPDATE = 0x00000002,
...
};
```
Unfortunately, I couldn't find a way to just make them resolve without the `GD_` prefix - maybe people have an idea how to do this within a macro.